storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteIndexCache.java

```diff
@@ -166,6 +166,11 @@ public Cache<Uuid, Entry> internalCache() {
         return internalCache;
     }
 
+    // Visible for testing
+    public File cacheDir() {
```

Comment:
Can we file a ticket to move the RemoteIndexCacheTest to `storage` module under the same package?

Reply:
@kamalcph  I have just started understanding of the kafka ecosystem , Can you help me understand the rational behind it , Then i can create a ticket with details.

Reply:
You can find more details about the intention to move the source code to Java in KAFKA-14524